openshift/charts/cluster-api-operator/templates/ocp-capa-provider.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    "argocd.argoproj.io/sync-wave": "1"
+    "helm.sh/hook": "post-install"
+    "helm.sh/hook-weight": "1"
+  name: capa-system
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  annotations:
+    "argocd.argoproj.io/sync-wave": "2"
+    "helm.sh/hook": "post-install"
+    "helm.sh/hook-weight": "2"
+  name: capa-manager-bootstrap-credentials
+  namespace: capa-system
+data:
+  # This is used to help encode your environment variables
+  # export AWS_REGION=us-east-1 
+  # export AWS_ACCESS_KEY_ID=<your-access-key>
+  # export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+  # export AWS_SESSION_TOKEN=<session-token> # If you are using Multi-Factor Auth.
+  # Create the base64 encoded credentials using clusterawsadm.
+  # This command uses your environment variables and encodes
+  # them in a value to be stored in a Kubernetes Secret.
+  # export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile|base64 -w0)
```

Comment:
I believe there is no need for the base64. check the doc ref here 
https://cluster-api-aws.sigs.k8s.io/clusterawsadm/clusterawsadm_bootstrap_credentials_encode-as-profile